src/cache/prognostic_edmf_precomputed_quantities.jl

```diff
@@ -174,16 +261,89 @@ NVTX.@annotate function set_prognostic_edmf_precomputed_quantities_bottom_bc!(
             obukhov_length_val,
             sfc_local_geometry_val,
         )
+        if p.atmos.moisture_model isa NonEquilMoistModel &&
+           p.atmos.precip_model isa Microphysics1Moment
+            ᶜq_liq_int_val =
+                Fields.field_values(Fields.level(ᶜspecific.q_liq, 1))
+            ᶜq_liqʲ_int_val = Fields.field_values(Fields.level(ᶜq_liqʲ, 1))
+            @. ᶜq_liqʲ_int_val = sgs_scalar_first_interior_bc(
+                ᶜz_int_val - z_sfc_val,
+                ᶜρ_int_val,
+                ᶜaʲ_int_val,
+                ᶜq_liq_int_val,
+                buoyancy_flux_val,
+                ρ_flux_q_liq_val, #TODO - what is the correct value for it? (and below)
+                ustar_val,
+                obukhov_length_val,
+                sfc_local_geometry_val,
+            )
```

Comment:
```suggestion
ᶜq_liqʲ_int_val = ᶜq_liq_int_val
```
and same for other tracers. You can leave a TODO for changing this when we know what to do.